src/components/pages/HomePage.jsx

```diff
@@ -16,10 +16,10 @@ const HomePage = () => {
 
   return (
     <>
-      <h1>HOME PAGE</h1>
+      <h1 className="text-3xl font-spartan font-bold underline">HOME PAGE</h1>
       <button onClick={search}>Search classes</button>
       <button onClick={login}>Log In</button>
-      <button onClick={register}>Join</button>
+      <button className='btn-red' onClick={register}>Join</button>
```

Comment:
It looks like this is being inherited from a `layer`. This is not going to be necessary. 

Take a look at how this buttons are styled.

[https://tailwindui.com/components/application-ui/elements/buttons](url)